lib/inferno/config/boot/db.rb

```diff
@@ -9,7 +9,7 @@
 
     Sequel::Model.plugin :json_serializer
 
-    config_path = File.expand_path('database.yml', File.join(Dir.pwd, 'config'))
+    config_path = File.expand_path('../../../../config/database.yml', __dir__)
```

Comment:
This change prevents users from customizing the database configuration, which is a necessary feature. Also, the `config` directory is not included in the `inferno_core` gem.

Reply:
I reverted this line and added a new database config specifically for the evaluator that uses sqlite in memory.